### ISSUE TITLE
No scrolling after quick j-k

### ISSUE THREAD

**Issue by oriongonza** [philc/vimium#4491]
**Describe the bug**

No scrolling after pressing k j (stands still)
**To Reproduce**

1. press j
2. release j
3. press k

**Comment by UncleSnail** [comment]
I cannot reproduce this issue. What browser/website are you using? Is it happening for you in all browsers or all websites?

**Comment by UncleSnail** [comment]
> Chrome, all websites. It happens only in linux (nixos, wayland). In my windows machine there's a slight stutter of a fraction of a second when you do it and it keeps going. In my linux it just stops

Okay. I am also on Wayland Linux (Arch with Sway wm) and I cannot reproduce this issue in either Firefox or Chromium. I would guess it is likely an issue caused by something else in your setup, like another addon, or your key polling rate of your setup, etc. Some keyboards also do weird things with the key outputs instead of sending out the "direct" signals, so maybe try using a different keyboard and see if it's fixed.
You could also try running your Nix install script on another machine and seeing if you still have the problem or if it is just that machine being slow for some reason.
Unfortunately, unless I can find a consistent way to reproduce this I don't see how we can debug it to fix the problem, especially if the problem isn't actually in Vimium.

**Comment by gdh1995** [comment]
How about changing screen DPI to 100%?

**Comment by UncleSnail** [comment]
My dpi is 100% I believe. I would need to check to make sure, but I don't
know why this would effect anything. Does changing it make a difference for
you?

**Comment by Drednaught608** [comment]
This seems to happen for me when I do so in this order:
press j
press k
release j
Or
press k
press j
release k

This order can happen whenever you're switching between scrolling with j & k in quick succession. The releasing of the opposite scroll key results in the scrolling stopping no matter the direction you're moving.

**Comment by UncleSnail** [comment]
> This seems to happen for me when I do so in this order: press j press k release j Or press k press j release k
> 
> This order can happen whenever you're switching between scrolling with j & k in quick succession. The releasing of the opposite scroll key results in the scrolling stopping no matter the direction you're moving.

Yes, there is a short pause when switching directions, but I believe that is the correct functionality. It should stop until your key repeat timeout is up (how long your computer takes before holding a key makes it type that key a bunch of times quickly). For me everything seems to be working correctly. If for you the scrolling actually stops and stays stopped, make sure when you hold down a key that key eventually starts repeating (IE, you haven't turned off repeating on your system) and if repeating works and the scrolling stops, let me know.

**Comment by philc** [comment]
Great writeup everyone. This is a duplicate of #2857, and has been fixed by #4717.
